libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -264,6 +264,33 @@ const AP_Param::GroupInfo AP_TECS::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("PTCH_FF_K", 30, AP_TECS, _pitch_ff_k, 0.0),
 
+    // @Param: LAND_PTRIM
+    // @DisplayName: Pitch angle for level flight in landing confiuration
+    // @Description: This sets the pitch angle required to fly straight and level with flaps and gear in the landing configuration. It is used to calculate the lower pitch limit applied during landing up antil the flare. This can be set to the average value of the AOA.AOA log data taken from a landing approach.
+    // @Range: -10 15
+    // @Units: deg
+    // @Increment: 1
+    // @User: Advanced
+    AP_GROUPINFO("LAND_PTRIM", 31, AP_TECS, _land_pitch_trim, 0),
+
+    // @Param: FLARE_HGT
+    // @DisplayName: Flare holdoff height
+    // @Description: When height above ground is below this, the sink rate will be held at TECS_LAND_SINK. Use this to perform a hold-of manoeuvre when combined with small values for TECS_LAND_SINK.
+    // @Range: -10 15
+    // @Units: deg
+    // @Increment: 1
+    // @User: Advanced
+    AP_GROUPINFO("FLARE_HGT", 32, AP_TECS, _flare_holdoff_hgt, 1.0f),
+
+    // @Param: HDEM_TCONST
+    // @DisplayName: Height Demand Time Constant
+    // @Description: This sets the time constant of the low pass filter that is applied to the height demand input when bit 1 of TECS_OPTIONS is not selected.
+    // @Range: -1.0 5.0
```

Comment:
negative time constant?

Reply:
Whoops. I removed the - sign and also added some code to protect against someone entering a too small value that would destabilise the LPF that is applied to the height demand.